src/calculateRentalCost.js

```diff
@@ -26,9 +26,20 @@
  *
  * @return {number}
  */
+'strict';
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalRentalCost = days * 40;
+  const basicDiscount = 20;
+  const discount = 50;
+
+  if (days >= 3 && days < 7) {
+    return totalRentalCost - basicDiscount;
+  } else if (days >= 7) {
+    return totalRentalCost - discount;
+  } else {
+    return totalRentalCost;
+  }
```

Comment:
```suggestion
  if (days >= 3 && days < 7) {
    return totalRentalCost - basicDiscount;
  }
  
  if (days >= 7) {
    return totalRentalCost - discount;
  }
  
  return totalRentalCost;
```
There is no need to use `else if` as your code will not go further after return anyway)